app/models/rules/sealed_bid.rb

```diff
@@ -0,0 +1,53 @@
+module Rules
+  class SealedBid < Rules::Basic
+    def winning_bid
+      return Presenter::Bid::Null.new if auction.available?
```

Comment:
After a talk, we decided to defer that for another PR, but it makes sense to maybe have a NullBid and NullAuction for AR/model layers with corresponding presenters above them